src/repository.c

```diff
@@ -1784,7 +1784,12 @@ static int repo_init_structure(
 			default_template = true;
 		}
 
-		if (tdir) {
+		/* if tdir was the empty string, treat it like tdir was a path to an
```

Comment:
We usually avoid asymmetric comments. Instead, we start multiline comments with a sole "/*". Furthermore, the fisrt character should be upper-cased. E.g.

```
/*
 * If tdir…
 * …
 */
```

See e.g. [1] as a reference (except for C99-style "//" comments). 

[1]: http://lkml.iu.edu/hypermail/linux/kernel/1607.1/00627.html